mypy/plugins/common.py

```diff
@@ -19,7 +19,7 @@
     Var,
 )
 from mypy.plugin import CheckerPluginInterface, ClassDefContext, SemanticAnalyzerPluginInterface
-from mypy.semanal import ALLOW_INCOMPATIBLE_OVERRIDE, set_callable_name
+from mypy.semanal_shared import ALLOW_INCOMPATIBLE_OVERRIDE, set_callable_name
```

Comment:
this move was needed to avoid a cyclic dependency between `semanal.py` and `plugins/common.py`